src/components/ErrorComponent.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { ErrorMessages } from './PeoplePage';
+
+interface ErrorComponentProps {
+  message: ErrorMessages;
+}
+
+export function ErrorComponent({ message }: ErrorComponentProps) {
+  return (
+    <>
+      {message === ErrorMessages.SOMETHING_WENT_WRONG && (
```

Comment:
The way you implemented this isn't wrong, but i would recommend to use dynamic property access to avoid repetition (and personally i think that's also improves code readability)!

Imagine if we have 50 types of error messages, what would we do?


an example implementation:

```js
    <>
        {message && (
            <p data-cy={`${message}Message`}>
            {ErrorMessages[message]}
            </p>
        )}
    </>
```